feature-libs/product-configurator/rulebased/components/price/configurator-price.component.ts

```diff
@@ -22,6 +28,38 @@ export interface ConfiguratorPriceComponentOptions {
 export class ConfiguratorPriceComponent {
   @Input() formula: ConfiguratorPriceComponentOptions;
 
+  //TODO(CXSPA-1014): make DirectionService a required dependency
+  constructor(
+    // eslint-disable-next-line @typescript-eslint/unified-signatures
+    directionService: DirectionService
+  );
+  constructor(
+    @Optional()
+    protected directionService?: DirectionService
+  ) {}
+
+  protected isRTLDirection(): boolean {
+    return this.directionService?.getDirection() === DirectionMode.RTL;
+  }
+
+  protected removeSign(value: string | undefined, sign: string): string {
+    if (value) {
+      return value.replace(sign, '');
+    }
+    return '';
+  }
+
+  protected addSing(
+    value: string | undefined,
+    sign: string,
+    before: boolean
+  ): string {
+    if (value) {
+      return before ? sign + value : value + sign;
+    }
+    return '';
+  }
```

Comment:
can we have just one method, that gets all required params:
- formattedValue
- value
- isRTL

and the returns the updated formatted Value.
So the method can then check internally, whether its a positive or a negative value and then add or remove the sign as necessary. So we have all the logic in one place.